docs/sdks/go/index.md

```diff
@@ -41,6 +41,7 @@ The source code can be found on GitHub: [momentohq/client-sdk-go](https://github
 
 - [Getting started with Momento Cache in Go](./cache.mdx)
 - [Getting started with Momento Topics in Go](./topics.mdx)
+- [Getting started with Momento Storage in Go](./storage.mdx)
```

Comment:
Can you please go ahead and add the storage related keywords from `storage.mdx` below into this file as well?